ramls/marccat.raml

```diff
@@ -16,6 +16,7 @@ types:
   searchResultVertical: !include schema/searchResultVertical.json
   searchResultCollection: !include schema/searchResultCollection.json
   authority: !include schema/authority.json
+  select: !include schema/select.json
```

Comment:
Can we use a more descriptive name than `select.json`? I see selecting as something you do from the UI, but this API is simply returning MARC authority info and the schema should reflect that.

Reply:
I have changed the name for `selectAuthorityHeader008Tag.json`